pallets/amm/src/weights.rs

```diff
@@ -27,15 +27,13 @@
 // --chain=dev
 // --steps=50
 // --repeat=20
-// --pallet
-// amm
-// --extrinsic
-// *
+// --pallet=amm
+// --extrinsic=*
 // --execution=wasm
 // --wasm-execution=compiled
 // --heap-pages=4096
-// --output=./pallets/amm/src/weights.rs
```

Comment:
This OK?

Reply:
yes. output is a destination where new weights are written. In case of amm - it is exactly there.